spec/unit/relation_spec.rb

```diff
@@ -13,18 +13,18 @@
 
   describe '#order' do
     it 'sorts documents' do
-      expect(users.order(name: :asc).only(:name).without(:_id).to_a.map(&:to_h)).
+      expect(users.order(name: Mongo::Index::ASCENDING).only(:name).without(:_id).to_a.map(&:to_h)).
         to eql([{name: 'Jane',}, {name: 'Joe'}])
 
-      expect(users.order(name: :desc).only(:name).without(:_id).to_a.map(&:to_h)).
+      expect(users.order(name: Mongo::Index::DESCENDING).only(:name).without(:_id).to_a.map(&:to_h)).
```

Comment:
it'd be better if we hide this behind our API, so basically `Mongo::Dataset#order` could translate `:desc` and `:asc` to corresponding driver constants